GUI.py

```diff
@@ -386,8 +510,9 @@ def __init__(self, myui):
     def run(self):
         self.ui.successfit = False
         try:
-            self.ui.p, self.ui.para, self.ui.para_names = fit(self.ui.x, self.ui.y, mod=self.ui.fitmod)
+            self.ui.p, self.ui.para, self.ui.para_names = fit(self.ui.x.squeeze(), self.ui.y.squeeze(), mod=self.ui.fitmod)

```

Comment:
Squeeze appears necessary when autofit is disabled, I believe the return of a manual fit is not a 1D array as expected, this keeps it generic as a hot fix but should be investigated further